locales/en-US.yml

```
@@ -832,6 +832,15 @@ reloadToApplySetting: "This setting will only apply after a page reload. Reload
 needReloadToApply: "A reload is required for this to be reflected."
 showTitlebar: "Show title bar"
 clearCache: "Clear cache"
+timelineCache: "Timeline cache"
+timelineCacheDescription: "Purge cache for your own timelines, lists, or antennas."
+purgeHomeTimelineCache: "Purge home timeline cache"
+purgeUserTimelineCache: "Purge user timeline cache"
+selectUserList: "Select a list"
+purgeUserListTimelineCache: "Purge list timeline cache"
+selectAntenna: "Select an antenna"
```

YAMLlint (1.37.1)

[error] duplication of key "selectAntenna" in mapping

(key-duplicates)